druid-shell/src/application.rs

```diff
@@ -75,9 +75,9 @@ impl Application {
     ///
     /// [druid#771]: https://github.com/linebender/druid/issues/771
     pub fn new() -> Result<Application, Error> {
-        if APPLICATION_CREATED.compare_and_swap(false, true, Ordering::AcqRel) {
-            return Err(Error::ApplicationAlreadyExists);
-        }
+        APPLICATION_CREATED
+            .compare_exchange(false, true, Ordering::SeqCst, Ordering::Acquire)
```

Comment:
I think the success ordering should probably remain `AcqRel`; `SeqCst` is an unneeded strengthening.

Reply:
Sure, on your say-so. This was me getting frustrated and going nuclear.